app/src/main/java/org/oppia/app/administratorcontrols/appversion/AppVersionViewModel.kt

```diff
@@ -19,7 +19,10 @@ class AppVersionViewModel @Inject constructor(
   val versionName = ObservableField<String>(BuildConfig.VERSION_NAME)
 
   private val lastUpdateDateTime =
-    fragment.activity!!.packageManager.getPackageInfo(fragment.activity!!.packageName, /* flags= */ 0).lastUpdateTime
+    fragment.activity!!.packageManager.getPackageInfo(
+      fragment.activity!!.packageName, /* flags= */
```

Comment:
Nit Change:
```
    fragment.activity!!.packageManager.getPackageInfo(
      fragment.activity!!.packageName,
       /* flags= */ 0
    ).lastUpdateTime
```

Reply:
ok, this should be on same line, I will update